contrib/julia-mode.el

```diff
@@ -131,6 +131,23 @@ This function provides equivalent functionality, but makes no efforts to optimis
              "\\\\"))
         (group "'"))))
 
+(defconst julia-hanging-assignment-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("=" "+=" "-=" "*=" "/=" "//=" "\\=" "^=" "÷=" "%=" "|=" "&=" "$=" "<<=" ">>="
+                        ".+=" ".-=" ".*=" "./=" ".//=" ".\\=" ".^=" ".÷=" ".%="))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
+
+(defconst julia-hanging-operator-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("^" ".^" "//" ".//" "*" "/" "%" "&" "\\" ".*" "./" ".%" ".\\"
+                        "<<" ">>" ">>>" ".<<" ".>>" ".>>>"
+                        "+" "-" "|" "$" ".+" ".-"
+                        ":" ".." "|>"
+                        ">" "<" ">=" "<=" "==" "===" "!=" "!==" "<:" "≠" "≤" "≥"
+                        ".>" ".<" ".>=" ".<=" ".==" ".!="
+                        "&&" "||" "?"))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
```

Comment:
Oh yes, I see there's some that are missing. I used [this table](http://docs.julialang.org/en/release-0.4/manual/mathematical-operations/#operator-precedence) because I saw [this comment](https://github.com/JuliaLang/julia/blob/7c4c123d6db4448a1628a8e2d0326761cf39472f/src/julia-parser.scm#L3) and wasn't sure how to interpret it for the purposes of julia-mode. 

By the way, do you think it's necessary to treat assignment operators differently? I was trying to preserve the original behavior with them, but I wasn't sure if that was important.